keras_nlp/models/roberta/roberta_preprocessor.py

```diff
@@ -152,6 +129,35 @@ class RobertaPreprocessor(keras.layers.Layer):
         lambda s1, s2: preprocessor(x=(s1, s2)),
         num_parallel_calls=tf.data.AUTOTUNE,
     )
+
+    # Alternatively, you can create a preprocessor from your own vocabulary.
+    # The usage is exactly the same as above.
+    vocab = {
```

Comment:
I might trim this down a bit for brevity. Just show the special tokens and "Ġafter", "noon", "Ġsun" or something like that. Same with the merges.